### PR TITLE
Fix: Remove extra asterisks

### PR DESCRIPTION
**Pull request by machikoyasuda**
noticed this bug on dev:
image

the fix:
image

### CODE REVIEW

File: benefits/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -274,16 +274,16 @@ msgstr ""
 msgid "core.pages.help.foss.link"
 msgstr "GitHub"
 
+msgid "core.buttons.previous_page"
+msgstr "Previous Page"
+
```
Comment (angela-tran) [inline benefits/locale/en/LC_MESSAGES/django.po:279]: I'm wondering why the script moved all these other entries around.

Comment (machikoyasuda) [reply benefits/locale/en/LC_MESSAGES/django.po:279]: It's so mysterious. It always moves these.

**Review by angela-tran — DISMISSED** [review]

**Comment by machikoyasuda** [comment]
@angela-tran Needs a re-review after rebasing, unfortunately!

**Review by angela-tran — APPROVED** [review]

**Comment by angela-tran** [comment]
@machikoyasuda Some small merge conflicts to resolve

**Comment by machikoyasuda** [comment]
@angela-tran 😅

**Review by thekaveman — APPROVED** [review]

**Pull request merged**